lib/internal/assert/calltracker.js

```diff
@@ -46,20 +47,22 @@ class CallTracker {
     const callChecks = this.#callChecks;
     callChecks.add(context);
 
-    return function() {
-      context.actual++;
-      if (context.actual === context.exact) {
-        // Once function has reached its call count remove it from
-        // callChecks set to prevent memory leaks.
-        callChecks.delete(context);
+    return new Proxy(fn, {
+      apply() {
```

Comment:
We need a null prototype object, otherwise the proxy will use the value of `Object.prototype.get` when it's set from userland.
```suggestion
      __proto__: null,
      apply() {
```

Reply:
Or can we also use `get` in proxy handler?

Reply:
__proto__ is better, there are other Object methods

Reply:
It seems that I still didn't understand the use of `Object.prototype.get` in test before, but now I understand,  thank you very much for the review.